src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.address.logic.commands;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.fail;
+import static seedu.address.testutil.TypicalJournalEntries.getTypicalJournal;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.text.ParseException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.AddAppointmentParser;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Appointment.Appointment;
+
+//@@author chenxing1992
+public class AddAppointmentCommandTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+
+    @Test
+    public void equals() throws ParseException, seedu.address.logic.parser.exceptions.ParseException {
+        String arg = "Lunch, tomorrow 5pm";
+
+        AddAppointmentCommand command = new AddAppointmentCommand(Index.fromOneBased(1), setAppointment(arg));
+        AddAppointmentCommand command2 = new AddAppointmentCommand(Index.fromOneBased(1), setAppointment(arg));
```

Comment:
your command can name expected command and actual command
AddAppointmentCommand expectedCommand = new AddAppointmentCommand(Index.fromOneBased(1), _**new Appointment(...)**_); 

instead of just copying the code for command and command2